code/__defines/urist/maths.dm

```diff
@@ -0,0 +1 @@
+#define ROUND_UP(x) ( -round(-(x)))
```

Comment:
unrelated to this pr but we should probably move the things in uristdefines.dm up to this folder.